#20 Graphical View Test File

GurkomalSRao opened the pull request:
Test file that validates if a properly contructed graphical view will be
displayed. The URL will validate if proper data is being trasmitted.
Validated using response codes (200 for correct 400 for incorrect)

Created Sample file for testing UI. Actual implementation will be added
afterwards

GurkomalSRao committed:
#20 Graphical View Test File

Test file that validates if a properly contructed graphical view will be
displayed. The URL will validate if proper data is being trasmitted.
Validated using response codes (200 for correct 400 for incorrect)

Created Sample file for testing UI. Actual implementation will be added
afterwards

Nick9500 commented on src/test/java/nitezh/ministock/UITesting/SimpleUITesting.java:
```diff
@@ -0,0 +1,23 @@
+
+    @Before
+    public void setup(){
+
```
Please remove whitespace.

GurkomalSRao replied:
corrected

Nick9500 commented on src/test/java/nitezh/ministock/dataaccess/ChartDataTest.java:
```diff
@@ -0,0 +1,109 @@
+                chdVars = chdVars+"%2C"+var[i];        //middle & end
+        }
+        urlString = "https://image-charts.com/chart?"+cht+"&"+chd;
+
```
Please remove extra line.

GurkomalSRao replied:
corrected

Nick9500 commented on src/test/java/nitezh/ministock/dataaccess/ChartDataTest.java:
```diff
@@ -0,0 +1,109 @@
+            assertEquals(request.HTTP_OK, request.getResponseCode());
+
+        }
+        catch( Exception e ){
```
Why not catch MalformedURLException?

GurkomalSRao replied:
url.open and request.connect() requires IOException rather than MalformedURLException. gives errors if MalformedURLException is used.

Nick9500 replied:
okay

jeffersoncasimir reviewed (changes requested):
In generateFailingGraphTest(), please address on line 98 where the User-Agent is set. This is a very minor issue, but only one User-Agent should be necessary.

GurkomalSRao committed:
#41 Fixed what was requested by reviewers

minor fixes requested by reviewers

GurkomalSRao committed:
#39 UIAutomator implementation

UIAutomator implementatation version 1.0

- no compliation errors when importing UIAutomator
- Issue: Does not recognize uiautomation test within test file

gmacrodose committed:
Test passes and visible

GurkomalSRao committed:
#39 UI Automator Tests

UI Automator version 1.1

-successfully compiles and completes tests
- minor issue acquiring values of auto complete drop down list.

TODO
Be able to select the first value of the autocomplete list

GurkomalSRao committed:
Seperate test

jeffersoncasimir reviewed (approved):
My requested change doesn't really affect anything or matter and I approve of the other requested changes.
